config/parameters.php

```diff
@@ -19,8 +19,8 @@
 
     // parallel
     $parameters->set(Option::PARALLEL, false);
-    $parameters->set(Option::PARALLEL_MAX_NUMBER_OF_PROCESSES, 20);
-    $parameters->set(Option::PARALLEL_JOB_SIZE, 60);
+    $parameters->set(Option::PARALLEL_MAX_NUMBER_OF_PROCESSES, 16);
+    $parameters->set(Option::PARALLEL_JOB_SIZE, 20);
```

Comment:
@TomasVotruba is the `Option::PARALLEL` need to be set to `true` ?

Reply:
Yes, it is `true` in our `rector.php` - https://github.com/rectorphp/rector-src/blob/fa7249028c579f405ed61704b215c87a417b97f5/rector.php#L64

It will take few months before it will become default, as we need to test is deeply in various OS.